draft-ietf-rats-corim.md

```diff
@@ -303,6 +303,17 @@ convention - e.g., {{Section 4.4.1.5 of -psa-token}}).
 {::include cddl/digest.cddl}
 ~~~
 
+### Tagged Bytes Type {#sec-common-tagged-bytes}
+
+An opaque, variable-length byte string.
+It can be used in different contexts: as an instance, class or group identifier in an `environment-map`; as a raw value measurement in a `measurement-values-map`.
+Its semantics is defined by the context in which it is found, and by the overarching CoRIM profile.
+When used as an identifier the responsible allocator entity SHOULD ensure uniqueness.
```

Comment:
the scope of uniqueness should be defined.

Reply:
You mean something like:

> _"[...] ensure uniqueness within the scope of the usage scope."_

Reply:
> You mean something like:
> 
> > _"[...] ensure uniqueness within the usage scope."_

OK

Reply:
cool, see [9757398](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/188/commits/9757398627525052f2b0bf3d2dd2c36affd790a8)